src/pip/_internal/wheel_builder.py

```diff
@@ -197,6 +197,15 @@ def _build_one_inside_env(
     global_options,  # type: List[str]
 ):
     # type: (...) -> Optional[str]
+
+    # Extend the list of global and build options passed on to
+    # the setup.py bdist_wheel call with the ones from the requirements file.
+    # Options specified in requirements file override those
+    # specified on the command line, since the last option given
+    # to setup.py is the one that is used.
+    global_options = global_options + req.global_options
+    build_options = build_options + req.build_options
```

Comment:
Does this have backward compatibility implications ?

Reply:
I don't think so?